scripts/powershell/sub-cleanup/sub-cleanup.ps1

```diff
@@ -0,0 +1,71 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Requires PowerShell modules Az and RemoteSigned excecution policy
+#
+# Commands to make this happen:
+#   Install-Module -Name Az -AllowClobber
+#   Set-ExecutionPolicy RemoteSigned
+
+function Print-Help 
+{
+    Write-Host @"
+====================== sub-cleanup.ps1 ======================
+Cleans up non-compliant resource groups in a specified Azure
+subscription.  Complient resource groups are named with a
+valid Microsoft alias or using the format: alias-<identifier>
+
+Usage:
+    sub-cleanup.ps1 <subscription Id> <exception file>
+
+Notes:
+    The optional exception file is a text file listing
+    resource groups to be skipped in the cleanup.
+=============================================================
+"@
+}
+
+if ($args.Count -lt 1)
+{
+    Print-Help
+}
+
+# Load exceptions file if one was provided
+$exceptions = @()
+if ($args.Count -eq 2)
+{
+    $exceptionFile = 
```

Comment:
Is this intended? if so what happens since there is nothing to the right of the equals sign?

Reply:
Nope.  Mistake.  I thought I cleaned it up.  I had thought that the CWD would need to be adjusted for the file's path, but it wasn't necessary.  It appears to always work when the path to the file is passed on the command line.